server/src/main/java/org/apache/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -74,6 +74,8 @@ public class CoordinatorDynamicConfig
   private final Map<Dimension, String> validDebugDimensions;
 
   private final Set<String> turboLoadingNodes;
+  private final Set<String> unmanagedNodes;
```

Comment:
Is this needed? Doesn't a server being in the value of the `cloneServers` automatically imply that the server is unmanaged?

Reply:
Removed `unmanagedNodes`, it should not be necessary.